src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -63,8 +63,16 @@
 
     private AgencyAndId stop;
 
+    /**
+     * The headsign of the alert
+     */
     private String direction;
```

Comment:
I'm not sure how many people actually use that API. I wanted to delete it at some point, but I never got around to asking if anyone was still using it. But of course it's always good to keep backwards compatibility in mind. :)